compose/compiler/compiler-hosted/src/main/java/androidx/compose/compiler/plugins/kotlin/lower/IrSourcePrinter.kt

```diff
@@ -783,7 +783,15 @@ private class IrSourcePrinterVisitor(
             }
             IrStatementOrigin.SAFE_CALL -> {
                 val lhs = expression.statements[0] as IrVariable
-                val rhs = expression.statements[1] as IrWhen
+                val rhsStatement = expression.statements[1]
+                val rhs = when (rhsStatement) {
+                    is IrBlock -> {
+                        rhsStatement.statements[1]
+                    }
+                    else -> {
+                        rhsStatement
+                    }
+                } as IrWhen
```

Comment:
is this change still needed? or did some changes later make it unnecessary? If it's still needed, no worries, just checking.

Reply:
Yes, it is still needed. For the testSafeCall `x?.A()` , rhsStatement is an IrBlock. For the testLetWithoutComposableCalls `x?.let {
                if (it > 0) {
                  NA()
                }
                NA()
              }
              A()` , rhsStatement is IrWhenImpl.